README.md

```diff
@@ -178,6 +178,11 @@ Beamium now expose metrics about his usage:
 | beamium_push_http_status | sink, status | counter | Push response http status code   |
 | beamium_push_errors      | sink         | counter | Number of push error             |
 
+## Hot reload
+Beamium now support hot reloading of his configuration! There is no specific thing to do to enable this feature. Actually, this support all features excepted those in relation with the logger.
```

Comment:
A readme is not an announcement, 
`Beamium support hot reloading of this configuration.`

Reply:
supports?

Reply:
I correct the mistake and move the hot reload section into the configuration section